### ISSUE TITLE
Why Very Old .NET? Any update for .NET 6/7?

### ISSUE THREAD

**Issue by sangeethnandakumar** [razorpay/razorpay-dot-net#110]
Why no support for any new .NET implementations?
.NET 4 is far far older and runs over .NET Framework not on CoreCLR !!!

Any plans? @amitrazorpay

**Comment by ssswagatss** [comment]
Yes, This should be updated to the Dotnet Core platform to make sure that it is platform indepenedent.

**Comment by sunnyagg** [comment]
Closing this, 3.2.0 version now supports till .NET 8.0
